pom.xml

```diff
@@ -116,6 +116,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Why is this dependecy required?

Reply:
This dependency was added to enable Spark UI (for testing purpose), now get it removed.